src/System.Net.NetworkInformation/tests/FunctionalTests/IPInterfacePropertiesTest_Linux.cs

```diff
@@ -0,0 +1,179 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Net.Sockets;
+using System.Net.Test.Common;
+
+using Xunit;
+using Xunit.Abstractions;
+
+namespace System.Net.NetworkInformation.Tests
+{
+    [PlatformSpecific(PlatformID.Linux)]
+    public class IPInterfacePropertiesTest_Linux
+    {
+        private readonly ITestOutputHelper _log;
+
+        public IPInterfacePropertiesTest_Linux()
+        {
+            _log = TestLogging.GetInstance();
+        }
+
+        [Fact]
+        public void IPInfoTest_AccessAllProperties_NoErrors()
+        {
+            foreach (NetworkInterface nic in NetworkInterface.GetAllNetworkInterfaces())
```

Comment:
For most of them, no, I don't think we can assert that it will be non-empty, although if we can we definitely should. I think if a network interface is enabled, then we can probably assert that it contains at least one Unicast address, although I'm not sure if that's something that always holds up.